### PR TITLE
picom-trans: add page

### PR DESCRIPTION
**Pull request by flipsi**
- **Version of the command being documented (if known):** 
`picom-trans` doesn't support versions, `picom` itself has version **vgit-7e568**.  

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/picom-trans.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# picom-trans
+
+> Set window opacity.
+> More information: <https://github.com/yshui/picom>.
+
+- Set opacity of current window to 90 percent:
```
Comment (EmilySeville7cfg) [inline pages/common/picom-trans.md:6]:
```suggestion
- Set the current window opacity to a specific percent:
```

File: pages/common/picom-trans.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`picom-trans --current --opacity={{90}}`
+
+- Set opacity of window with name 'Firefox' to 90 percent:
```
Comment (EmilySeville7cfg) [inline pages/common/picom-trans.md:10]:
```suggestion
- Set a specific window opacity:
```

File: pages/common/picom-trans.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`picom-trans --name={{Firefox}} --opacity={{90}}`
+
+- Select window to set opacity for with mouse cursor:
```
Comment (EmilySeville7cfg) [inline pages/common/picom-trans.md:14]:
```suggestion
- Set an opacity for a specific window selected via mouse cursor:
```

File: pages/common/picom-trans.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`picom-trans --select --opacity={{90}}`
+
+- Toggle opacity for window with name 'Firefox':
```
Comment (EmilySeville7cfg) [inline pages/common/picom-trans.md:18]:
```suggestion
- Toggle a specific window opacity:
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/picom-trans.md (hunk 2 above)
Comment (navarroaxel) [reply pages/common/picom-trans.md:10]: Something like this? :thinking: 
```suggestion
- Set an opacity for a window with a specific name:
```

Comment (EmilySeville7cfg) [reply pages/common/picom-trans.md:10]: Yeah, sounds nicer than my variant.

Comment (marchersimon) [reply pages/common/picom-trans.md:10]: I'd use the definite article, however: `Set the opacity ...`

File: pages/common/picom-trans.md (hunk 4 above)
Comment (marchersimon) [reply pages/common/picom-trans.md:18]: I'm not entirely sure, but I think `Firefox` refers to a window class here, not a single window. Is this correct @flipsi?

File: pages/common/picom-trans.md (hunk 1 above)
Comment (marchersimon) [reply pages/common/picom-trans.md:6]: @flipsi I assume `current window` means `currently focused window`, right? It is a bit wordy, but I think we should at least clarify this in the first example.

Comment (flipsi) [reply pages/common/picom-trans.md]: > @flipsi I assume `current window` means `currently focused window`, right? It is a bit wordy, but I think we should at least clarify this in the first example.

Correct, thanks for pointing this out! 

I [adjusted the wording](https://github.com/tldr-pages/tldr/pull/8244/commits/8f97d5e862b4b9a68ba2ebdfd48938e85fe33f7e).

File: pages/common/picom-trans.md (hunk 4 above)
Comment (flipsi) [reply pages/common/picom-trans.md]: No, as far as I understand, window name is different from window class and this matches the name.
At least I tested this and compared my query with the different values I got from `xprop`. 
Window name is what is most likely shown as a title in window decorations of window managers.

**Comment by flipsi** [comment]
Thank you for your detailed review!

I didn't like the concrete values in descriptions either, but wanted to be consistent with what I found in `pages/common/split.md`. Anyway, I [rephrased the descriptions now].

File: pages/common/picom-trans.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# picom-trans
+
+> Set window opacity.
+> More information: <https://github.com/yshui/picom>.
+
+- Set the currently focused window opacity to a certain percentage:
```
Comment (EmilySeville7cfg) [inline pages/common/picom-trans.md:6]:
```suggestion
- Set the current window opacity to a specific percentage:
```

AFAIK we prefer `a` instead of `the`, and `specific` instead of `certain`. Pages should be standardized. ;)

File: pages/common/picom-trans.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`picom-trans --current --opacity={{90}}`
+
+- Set the opacity of a window with a specific name:
```
Comment (EmilySeville7cfg) [inline pages/common/picom-trans.md:10]:
```suggestion
- Set an opacity of a window with a specific name:
```

File: pages/common/picom-trans.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`picom-trans --name={{Firefox}} --opacity={{90}}`
+
+- Set the opacity of a specific window selected via mouse cursor:
```
Comment (EmilySeville7cfg) [inline pages/common/picom-trans.md:14]:
```suggestion
- Set an opacity of a specific window selected via mouse cursor:
```

File: pages/common/picom-trans.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+`picom-trans --select --opacity={{90}}`
+
+- Toggle the opacity of a specific window:
```
Comment (EmilySeville7cfg) [inline pages/common/picom-trans.md:18]:
```suggestion
- Toggle an opacity of a specific window:
```

**Review by EmilySeville7cfg — DISMISSED** [review]

File: pages/common/picom-trans.md (hunk 5 above)
Comment (marchersimon) [reply pages/common/picom-trans.md:6]: I agree with `specific`, but what's wrong with `currently focused`. It's more precise than `current`.

File: pages/common/picom-trans.md (hunk 6 above)
Comment (marchersimon) [reply pages/common/picom-trans.md:10]: I think `the` is better in this context, because every window has one opacity. And we're setting this very opacity. `An opacity` sounds like we're setting one of the many opacities a window could have.

File: pages/common/picom-trans.md (hunk 5 above)
Comment (flipsi) [reply pages/common/picom-trans.md:6]: 1) What about [Simon's](https://github.com/tldr-pages/tldr/pull/8244#discussion_r928295348) remark about "focussed"? I think we should incorporate this.
2) Okay, but I'm already using `a` (at least in the descriptions below). I think using an indefinite article twice doesn't make much sense here: I'm talking a about **a** window, so it's **the** opacity of this particular window (definite in this context).
3) Okay, I will change `certain` to `specific`. (Wanted to avoid using `specific` twice in the second description)

File: pages/common/picom-trans.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
+# picom-trans
+
+> Set window opacity.
```
Comment (marchersimon) [inline pages/common/picom-trans.md:3]:
```suggestion
> Set window opacity for the `picom` window compositor.
```

File: pages/common/picom-trans.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+> Set the window opacity for the `picom` window compositor.
+> More information: <https://github.com/yshui/picom>.
+
+- Set the currently focused window opacity to a specific percentage:
+
+`picom-trans --current --opacity={{90}}`
+
+- Set the opacity of a window with a specific name:
+
+`picom-trans --name={{Firefox}} --opacity={{90}}`
```
Comment (marchersimon) [inline pages/common/picom-trans.md:12]: Just one last thing from my side: Could you please test if those flags work without the `=` and remove them if they do?

Comment (flipsi) [reply pages/common/picom-trans.md]: Sure. They do work, so [I removed them as you requested](https://github.com/tldr-pages/tldr/pull/8244/commits/04d516ca26a97fd7bd6bd158dc1dbec86ad1a625). 

The man page however suggest using `=`. So just out of curiosity: Shouldn't we stick to that?

Comment (marchersimon) [reply pages/common/picom-trans.md]: Because it makes it more readable. And it's easier to just never use them instead of remembering where they work and where not. In the end it doesn't make a difference.

**Review by marchersimon — APPROVED** [review]
Thanks @flipsi

**Review by CleanMachine1 — APPROVED** [review]

**Comment by CleanMachine1** [comment]
Thanks for your contribution!
